microsoft-edge/privacy-whitepaper/index.md

```diff
@@ -190,15 +131,96 @@ You can access Collections from an InPrivate window when using a signed-out prof
 <!-- ====================================================================== -->
 ## Compose
 
-In the Copilot pane, the Compose tool uses AI to write for you, starting from a short prompt and **Tone**, **Format**, and **Length** selections. Text that's generated by Compose can be copied or inserted into the webpage directly.
+
+<!-- ------------------------------ -->
+#### Draft text by using Compose in the Edge Sidebar
+
+In the Copilot pane, the Compose tool uses AI to write for you, starting from a short prompt and **Tone**, **Format**, and **Length** selections:
 
 ![Compose UI](./index-images/compose.png)
 
-To access Compose, click the **Copilot** (![Copilot icon](./index-images/copilot-icon.png)) icon (**Ctrl+Shift+.**) in the upper right of Microsoft Edge, and then select **Compose**. Text and selections that you enter are sent to an internal Microsoft Large Language Model (LLM). 
+Text that's generated by Compose can be copied, or inserted directly into the webpage.
 
-Requests for Compose are sent over a secure HTTPS connection. Each request contains the input prompt, formatting selections, and IDs that are associated with your Microsoft Edge profile and device, to reference for subsequent requests.
+To access the Compose tool, in the upper right of Microsoft Edge, click the **Copilot** (![Copilot icon](./index-images/copilot-icon.png)) button (**Ctrl+Shift+.**), and then select **Compose**.  Text that you select or enter is sent to an internal Microsoft Large Language Model (LLM). 
+
+To change settings for the sidebar or Copilot pane, select **Settings and more** (![Settings and more icon](./index-images/settings-and-more-icon.png)) > **Sidebar** > **App and notification settings** > **Copilot**.
+
+
+<!-- ------------------------------ -->
+#### Rewrite selected text on any webpage by using Compose 
 
-To change settings for the sidebar or Copilot pane, go to `edge://settings/sidebar`, and then select **App and notification settings** > **Copilot**.
+The Compose tool can also help rephrase what you've written, and iterate using the same **Tone**, **Format**, and **Length** selections. 
```

Comment:
get rid of "also".  instead, in the h2 section / intro, list or mention both scenarios of Compose: eg:

The Compose tool enables you to draft text by using Compose in the Edge Sidebar, or rewrite selected text on any webpage by using Compose.

i can enter specific Suggested Change comments, or push a commit to impl. this

Reply:
```
## Compose
The Compose tool helps you draft text within the Microsoft Edge Sidebar, or rewrite selected text on any webpage.

#### Use Compose to draft text within the Edge Sidebar
In the Copilot pane, the Compose tool uses AI to write for you, starting from a short prompt and **Tone**, **Format**, and **Length** selections: ...

#### Use Compose to rewrite selected text on any webpage
The Compose tool can help rephrase what you've written, and iterate using the same **Tone**, **Format**, and **Length** selections. ...
```